### PR TITLE
support browser test type WebpageAction

### PR DESCRIPTION
**Pull request by swethalohith2**
## What

support browser test type WebpageAction

## References

- [Story](https://instana.kanbanize.com/ctrl_board/132/cards/132702/details/)

### CODE REVIEW

File: synctl [hunk 1]
```diff
@@ -590,10 +593,16 @@ class SyntheticConfiguration(Base):
             "browser": "chrome",
         }
 
-        # self.webpage_action_conf = {}
+        # syntheticType WebpageAction
+        self.webpage_action_conf = {
```
Comment (rongzhus) [inline synctl:597]: The variable `self.webpage_action_conf` is defined but never used.

**Comment by rongzhus** [comment]
@swethalohith2 I got another issue when I use synctl create command to create an API script. It shows 
```
synctl create: error: the following arguments are required: --record-video
```

My command is 
```
synctl create test -t 1 --label api-script-test -f script1.js --lo Mff4Rd7NxXDW2wQlAEug --frequency 5
```

File: synctl [hunk 2]
```diff
@@ -590,10 +593,15 @@ class SyntheticConfiguration(Base):
             "browser": "chrome",
         }
 
-        # self.webpage_action_conf = {}
+        #syntheticType WebpageAction
+        self.webpage_action_conf = {
+            "syntheticType": "WebpageAction",
+            "browser": "chrome",
+            "recordVideo": True
```
Comment (lijingmu) [inline synctl:600]: By default, `recordVideo` should be false. We can remove it as other test definition.

**Comment by lijingmu** [comment]
By default `"recordVideo":` is false. If users do not set it with Open API, it is false. Users can set it to true if then want to enable video recording for test types of `WebpageScript`, `WebpageAction`, `BrowserScript`.

**Comment by rongzhus** [comment]
@swethalohith2 One more command need to be enhanced to support WebpageAction:

```
synctl get test -t 4
```

**Review by rongzhus — APPROVED** [review]
LGTM. Need your @lijingmu confirm.

**Comment by lijingmu** [comment]
LGTM, thanks

**Pull request merged**